.gitignore

```
@@ -2,16 +2,9 @@
 /build/
 /.gradle/
 
-# intellij
```

I would recommend keeping these

---

I figure it's best to only have things that need to be ignored regardless of dev environment here, while anything user-specific should live in a global gitignore

---

It's a matter of preference, I guess, but I've always seen gitignores err on the side of caution in enterprise repos

---

I can see that being a thing in the enterprise because development environments tend to be a bit more uniform on smaller teams as opposed to public open source stuff where any yahoo can clone the repo and fool around